content/solana-nft.md

```diff
@@ -543,7 +543,7 @@ Refresh [localhost:3000](http://localhost:3000) in the browser and click the min
 
 Next, let's create a whitelist token.
 
-Metaplex restricts token creation, so let's set our connection to Devnet RPC:
+The Metaplex RPC restricts token creation, so let's set our connection to Devnet RPC:
```

Comment:
Unsure if RPC is the correct vocab to describe
RPC or endpoint?

Reply:
@ZYJLiu good question - does a remote procedure call determine the restriction, or is it the remote _**endpoint**_ being called?

Would it make the point any clearer if we removed "RPC" altogether for, "Metaplex retricts token...so let's set...to Devnet."?

Reply:
well they're both connecting to devnet, it's just that metaplex provides their own because the public one gets rate-limited 
although it seems that the metaplex connection restricts transactions with non-metaplex related instructions

Reply:
Not sure if going over the connection difference would be out of place in the overview since its was more-so just a workaround in order to be able to use the whitelist setting functionality in the demo. 

The reason the demo switches connection back and forth is because when I was working through the demo, the command to upload the candy machine wouldn't go through using the official Solana devnet RPC 

The flow of the demo is 
1) Connect using Metaplex RPC to demo candy machine without whitelist
2) Connect using Solana RPC to create whitelist token
3) Switch connection back to Metaplex RPC
4) Demo candy machine whitelistsetting with the token we just created